src/android/com/adobe/phonegap/push/GCMIntentService.java

```diff
@@ -430,7 +430,8 @@ private void createActions(Bundle extras, NotificationCompat.Builder mBuilder, R
                 JSONArray actionsArray = new JSONArray(actions);
                 ArrayList<NotificationCompat.Action> wActions = new ArrayList<NotificationCompat.Action>();
                 for (int i=0; i < actionsArray.length(); i++) {
-                    int uniquePendingIntentRequestCode = i + notId;
+                    // This effectively appends i's value to the notId to ensure uniqueness when multiple notId's are sequential
+                    int uniquePendingIntentRequestCode = notId * 10 + i;
```

Comment:
@mikepsinn could we get an integer overflow problem here? Why the need to multiply by 10?